backend/run/copr-backend-resultdir-cleaner

```diff
@@ -94,9 +94,7 @@ def clean_in(resultdir, dry_run=True):
                 continue
 
             if subdir in ["tmp", "cache", "appdata"]:
-                # tmp => 95k * 7 inodes
-                # cache => 95k * 2 inodes
-                todo_directory(chroot_subdir_path, "APPSTREAMFAIL")
+                remove_old_dir(chroot_subdir_path, dry_run)
```

Comment:
There's 150k of such directories;  I'd like to clean them once this is merged (hot-fix).